src/Content.tsx

```diff
@@ -2,27 +2,39 @@ import React from 'react';
 import { createRoot } from 'react-dom/client';
 import ProfessorPopup from './ProfessorPopup';
 
-const iframe = document.getElementById('ptifrmtgtframe') as HTMLIFrameElement;
+if (document.URL.includes('https://cmsweb.cms.cpp.edu/'))
+  console.log('BroncoDirect Page Loaded');
 
-// event listener on the iframe which fires on search
-// since iframe will receive a post message of the search criteria
-// however event will fire regardless if the search criteria is invalid and will remain on the page
-iframe?.contentWindow?.addEventListener('message', () => {
-  const insts: NodeListOf<HTMLElement> | undefined =
-    iframe.contentWindow?.document.querySelectorAll('*[id^="MTG_INSTR$"]');
-  // iterate through insts and create new instance of ProfessorPopup for each inst
-  insts?.forEach((inst) => {
-    // append a new root container under inst.parent to retain original span element
-    const root = document.createElement('div');
-    const parentElem = inst.parentElement as HTMLDivElement;
+if (
+  document.URL.includes(
+    'https://cmsweb.cms.cpp.edu/psp/CPOMPRDM/EMPLOYEE/SA/c/SA_LEARNER_SERVICES.CLASS_SEARCH.GBL?'
+  )
+) {
+  const iframe = document.getElementById('ptifrmtgtframe') as HTMLIFrameElement;
 
-    // styling for the ProfessorPopup Button
-    root.setAttribute('id', 'root');
-    root.style.float = 'right';
-    parentElem?.append(root);
+  // event listener on the iframe which fires on search
+  // since iframe will receive a post message of the search criteria
+  // however event will fire regardless if the search criteria is invalid and will remain on the page
+  iframe?.contentWindow?.addEventListener('message', () => {
+    const insts: NodeListOf<HTMLElement> | undefined =
+      iframe.contentWindow?.document.querySelectorAll('*[id^="MTG_INSTR$"]');
+    if (insts?.length == 0) console.log('- Enter Search Criteria Page');
```

Comment:
```suggestion
    if (insts?.length === 0) console.log('- Enter Search Criteria Page');
```
it should be triple equals for the linter.